dali/kernels/test/mat2tensor.h

```diff
@@ -28,15 +28,15 @@ TensorShape<ndim> tensor_shape(const cv::Mat &mat) {
   TensorShape<ndim> shape;
   if (ndim != DynamicDimensions) {
     // require extra dimension for channels, if #channels > 1
-    if (ndim != mat.size.dims() + 1 && !(ndim == mat.size.dims() && mat.channels() == 1))
+    if (ndim != mat.size[-1] + 1 && !(ndim == mat.size[-1] && mat.channels() == 1))
```

Comment:
curious, what does this `-1` mean

Reply:
Some cv::MatSize magic - https://github.com/opencv/opencv/blob/master/modules/core/include/opencv2/core/mat.inl.hpp#L1450. Now it is exposed by dims() function but previously it was not available directly.

Reply:
A comment here would be nice.

Reply:
Done